src/utils.py

```diff
@@ -20,7 +20,7 @@ def __init__(self):
             elif "Hardware Vendor" in line:
                 self.vender = line.strip().split(':', 1)[1]
             elif "Static hostname" in line:
-                self.hostname = line.strip().split(':', 1)[1]
+                self.hostname = line.strip().split(':', 1)[1].strip()
```

Comment:
@Phqen1x FYI this particular change was actually fixing a bug. The get_hostname function was returning the string with a leading space, which broke the startswith check.